src/components/Calendar.tsx

```diff
@@ -30,7 +30,7 @@ const StyledRow = styled.div<{ showRowCounter: boolean }>`
         width: 12px;
         font-size: 12px;
         margin: 0px 3px 6px 0px;
-        visibility: ${props => props.showRowCounter ? "visible" : "hidden"};
+        visibility: ${props => props.$showRowCounter ? "visible" : "hidden"};
```

Comment:
haven't really ever seen a dollar sign used in prop names, nothing wrong with it, just new haha

Reply:
Yea it's specific to styled components. If it's used without the `$`, the console will spit out an error/warning telling you that this prop is not defined/not found because it gets confused with the component's actual props in the DOM or something.